dominio/Escola.cs

```diff
@@ -0,0 +1,22 @@
+namespace dominio
+{
+    public class Escola
+    {
+        public int codigo_escola { get; set; }
+        public string nome_escola { get; set; }
+        public int Id_rede { get; set; }
+        public string CEP { get; set; }
+        public int Id_uf { get; set; }
+        public string Endereco { get; set; }
+        public int Id_municipio { get; set; }
+        public int Id_localizacao { get; set; }
+        public string Longitude { get; set; }
+        public string Latitude { get; set; }
+        public int Id_etapas_de_ensino { get; set;} 
+        public int Numero_total_de_alunos { get; set; }
+        public int Id_situacao { get; set; }
+        public int Id_porte { get; set; }
+        public string Telefone { get; set; }
+        public int Numero_total_de_docentes{ get; set; }
```

Comment:
Como agora vamos mapear os atributos pelo próprio comando SELECT que está no repositório, podem usar PascalCase aqui:
```suggestion
        public int IdEscola { get; set; }
        public int CodigoEscola { get; set; }
        public string NomeEscola { get; set; }
        public int IdRede { get; set; }
        public string Cep { get; set; }
        public int IdUf { get; set; }
        public string Endereco { get; set; }
        public int IdMunicipio { get; set; }
        public int IdLocalizacao { get; set; }
        public string Longitude { get; set; }
        public string Latitude { get; set; }
        public int IdEtapasDeEnsino { get; set;} 
        public int NumeroTotalDeAlunos { get; set; }
        public int IdSituacao { get; set; }
        public int IdPorte { get; set; }
        public string Telefone { get; set; }
        public int NumeroTotalDeDocentes{ get; set; }
```